<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #FAFAFA;
}</string>
  </property>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>145</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt;
color: #444444;
background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>235</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt;
color: #444444;
background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: #71cd00;
	border-radius: 10px;
	font: 16pt, arial;
	color: #ffffff;
}
QPushButton-Hover:
{
	background-color: #549900;
}
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>281</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 2px solid #71cd00;
	border-radius: 10px;
}
QLineEdit:focus
{
	border: 2px solid #71cd00;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>281</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 2px solid #71cd00;
	border-radius: 10px;
}
QLineEdit:focus
{
	border: 2px solid #71cd00;
}</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="remembermeCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>Remember me?</string>
   </property>
  </widget>
  <widget class="QLabel" name="orLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>--------------------------- OR ---------------------------</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="needanaccountLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>Need an account?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="signupPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: None;
	text-decoration: underline;
	color: #1c33de;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgotpasswordPushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border: None;
	text-decoration: underline;
	color: #aa1111;
	background-color: rgba(0,0,0,0);
}
QPushButton:hover{
	color: #630b0b;
}</string>
   </property>
   <property name="text">
    <string>Forgot Password?</string>
   </property>
  </widget>
  <widget class="QLabel" name="invalidemailLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #aa1111;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>*Invalid Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="invalidpasswordLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #aa1111;
	background-color: rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string>*Invalid Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt, Arial;
color: #444444;
background-color: rgba(0,0,0,0);</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edockiconPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>23</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/edock.png</normaloff>images/edock.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <zorder>invalidemailLabel</zorder>
  <zorder>invalidpasswordLabel</zorder>
  <zorder>emailLabel</zorder>
  <zorder>passwordLabel</zorder>
  <zorder>loginPushButton</zorder>
  <zorder>remembermeCheckBox</zorder>
  <zorder>orLabel</zorder>
  <zorder>needanaccountLabel</zorder>
  <zorder>signupPushButton</zorder>
  <zorder>forgotpasswordPushButton</zorder>
  <zorder>emailLineEdit</zorder>
  <zorder>passwordLineEdit</zorder>
  <zorder>loginLabel</zorder>
  <zorder>edockiconPushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
